packages/components/src/chart/d3chart/chart.js

```diff
@@ -124,6 +128,11 @@ class D3Chart extends Component {
 		const xLineScale = getXLineScale( uniqueDates, adjWidth );
 		const xScale = getXScale( uniqueDates, adjWidth, compact );
 		const xTicks = getXTicks( uniqueDates, adjWidth, mode, interval );
+
+		this.setState( {
+			isEmpty: yMax === 0,
```

Comment:
I know this is likely to catch all instances for our purposes, but since we are releasing `<Chart />` as a package, it could be used for display of entirely negative numbers. Is there another check that may be more explicit?

Reply:
Good point. Currently the `<Chart />` component is not prepared to display negative values, but if at some point we want to support it, it makes sense to implement new features with that in mind.

I refactored completely the way we check whether the chart is empty in c24bbba226b2112f859fcea98afb9e31b9040e3f. This way, I could also remove the state from the component.